utils/build_system/create_test_release_jobs.py

```diff
@@ -34,5 +37,31 @@ def create_pipeline_job(self, jenkins_file, group_name, job_name=None, job_name_
                 group_name = "/" + group_name
             self.jenkins.create_job(
                 f'{self.base_job_dir}{group_name}/{base_name}{job_name_suffix}', xml_data)
+            self.build_job_first_time(f'{self.base_job_dir}{group_name}/{base_name}{job_name_suffix}')
         except jenkins.JenkinsException as ex:
             print(ex)
+
+    def build_job_first_time(self, name):
```

Comment:
Technically, this method is not tied to "first" build of a job. So, we should remove word "first" from the name.
I propose to name it "build_job_and_wait_for_completion".

Reply:
This method will called only if job is created and run first time, thats was a reason why it called like that. 
I will rename.